node/src/OpenTelemetry.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ *
+ * ⚠️ OpenTelemetry can only be initialized once per process. Calling `OpenTelemetry.init()` more than once will be ignored.
+ * If you need to change configuration, restart the process with new settings.
+ *
+ * ### OpenTelemetry
+ *
+ * - **openTelemetryConfig**: Use this object to configure OpenTelemetry exporters and options.
+ *   - **traces**: (optional) Configure trace exporting.
+ *     - **endpoint**: The collector endpoint for traces. Supported protocols:
+ *       - `http://` or `https://` for HTTP/HTTPS
+ *       - `grpc://` for gRPC
+ *       - `file://` for local file export (see below)
+ *     - **samplePercentage**: (optional) The percentage of requests to sample and create a span for, used to measure command duration. Must be between 0 and 100. Defaults to 1 if not specified.
```

Comment:
we can have the notice here - about the tradeoff between sampling to performance